doc/compatibility_matrix.rst

```diff
@@ -10,39 +10,53 @@ The following table demonstrates the compatibility between the nRF Connect SDK a
    :widths: auto
    :header-rows: 1
 
-   * - nRF Connect SDK version
+   * - Add-on version
+     - nRF Connect SDK version
      - Amazon specification (Sidewalk MCU SDK version)
      - Supported platforms
      - The Amazon Sidewalk Protocol Implementation Conformance Statement (PICS)
      - Amazon qualification status
-   * - v2.9.0
+   * - v0.1.0
+     - v3.0.0
+     - #######TBA
+     - * `nRF52840 DK`_
+       * `nRF5340 DK`_
+       * `nRF54L15 DK`_
+     - #######TBA
+     - #######TBA

```

Comment:
There is no release 3.0.0, and there will be no release 3.0.0

Reply:
My understanding as to what we should document, was that each addon release is based on a certain ncs release, which should be indicated in the compatibility table. If my understanding is correct, then 3.0.0 certainly does exist. We could maybe rename the heading

Reply:
FYI: agreed with @wojciechpluszynski to leave the NCS placeholder for the next release and keep it as N/A for the initial one